company-org-roam.el

```diff
@@ -89,7 +88,7 @@ The string match is case-insensitive."
 Entries with no title do not appear in the completions."
   (let ((dir (file-truename org-roam-directory))
         (ht (make-hash-table :test #'equal)))
-    (dolist (row (org-roam-sql [:select [titles file] :from titles]))
+    (dolist (row (org-roam-db-query [:select [titles file] :from titles]))
```

Comment:
One more thing, can you alias this as well? I'm using it in `company-org-roam`.

Reply:
Taken care of